finance_manager/src/test/java/com/manager/finance/service/wallet/WalletServiceTest.java

```diff
@@ -0,0 +1,161 @@
+package com.manager.finance.service.wallet;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.manager.finance.exception.wallet.WalletNotFoundException;
+import com.manager.finance.util.TimeZoneUtils;
+import com.manager.finance.wallet.dao.WalletDao;
+import com.manager.finance.wallet.model.entity.Wallet;
+import com.manager.finance.wallet.model.entity.api.DefaultCurrency;
+import com.manager.finance.wallet.service.api.WalletService;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.test.context.ActiveProfiles;
+
+@ActiveProfiles("dev")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class WalletServiceTest {
+    private static final List<Wallet> wallets;
+
+    static {
```

Comment:
Please consider either static block or @BeforeAll. 
Seems we can have one of them only.